src/System/SshdLint/Check.hs

```diff
@@ -64,7 +68,7 @@ recommendations recommendedSettings settings =
                   normalizedSettings
 
   where normalizedRecommendedSettings = lowerCaseMapKeys recommendedSettings
-        normalizedSettings = lowerCaseMapKeys settings
+        normalizedSettings = Map.map (flip (:) []) $ lowerCaseMapKeys settings
```

Comment:
@juanpaucar Would it be clearer if you replace `flip (:) []` with either `return` or `pure`?